gateway/protoc-gen-cc-gateway/generator/template.go

```diff
@@ -118,9 +120,13 @@ type {{ $svc.GetName }}Gateway struct {
 	ChaincodeInstance cckit_gateway.ChaincodeInstance
 }
 
-
 func (c *{{ $svc.GetName }}Gateway) Invoker() cckit_gateway.ChaincodeInstanceInvoker {
-   return cckit_gateway.NewChaincodeInstanceServiceInvoker(c.ChaincodeInstance)
+   invoker := cckit_gateway.NewChaincodeInstanceServiceInvoker(c.ChaincodeInstance)
```

Comment:
suggest to add possibility to set invoker  after gateway creation

this will allow to set invoker with envelope and don't couple generated gateway with envelope